modules/ztp-precaching-downloading-artifacts.adoc

```diff
@@ -98,7 +98,7 @@ The {factory-prestaging-tool} allows you to pre-cache all the container images r
 ----
 # podman run -v /mnt:/mnt -v /root/.docker:/root/.docker --privileged --rm quay.io/openshift-kni/telco-ran-tools -- \
    factory-precaching-cli download \ <1>
-   -r 4.11.5 \ <2>
```

Comment:
@amolnar-rh need to verify this with QE

Reply:
hi @amolnar-rh, do you need who we need to tag to verify this?

Reply:
@alosadagrande Could you please take a look asap?

Reply:
For 4.12.0 there is no channel stable-2.0 for MCE either release-2.5 for ACM. So, replace:

--acm-version 2.6.3
--mce-version 2.0.4

You'll need to update the rest of the doc to match those new args.